etc/gather_coverage.jl

```diff
@@ -6,9 +6,7 @@ using Coverage
 import JSON
 
 # gather coverage
-data = Codecov.process_folder("pkg/GAPJulia/JuliaInterface")
-append!(data, Codecov.process_folder("pkg/GAPJulia/JuliaExperimental"))
-append!(data, Codecov.process_folder())
+data = Codecov.process_folder("$(ENV["HOME"])")
```

Comment:
That string quoting looks wrong

Reply:
Nope, that string quoting is correct in Julia.